src/content/containers/Home.js

```diff
@@ -148,6 +169,8 @@ class Home extends PureComponent {
             <span className='homeSectionPlus'>+</span>
             Install add-on
           </InstallButton>
+          <p>Upload a saved profile</p>
```

Comment:
I'd have a sentence a little more user-friendly, like `You can also directly upload a profile either by drag-and-dropping a file here, or using the following button:`